# Description: Change overlay scrollbar type to the fading type used on Android
# Author: Chris Coulson <chris.coulson@canonical.com>

diff --git a/content/renderer/gpu/render_widget_compositor.cc b/content/renderer/gpu/render_widget_compositor.cc
--- a/content/renderer/gpu/render_widget_compositor.cc
+++ b/content/renderer/gpu/render_widget_compositor.cc
@@ -259,17 +259,17 @@ scoped_ptr<RenderWidgetCompositor> Rende
   // RGBA_4444 textures are only enabled for low end devices
   // and are disabled for Android WebView as it doesn't support the format.
   settings.use_rgba_4444_textures =
       base::android::SysUtils::IsLowEndDevice() &&
       !widget->UsingSynchronousRendererCompositor() &&
       !cmd->HasSwitch(cc::switches::kDisable4444Textures);
 #elif !defined(OS_MACOSX)
   if (IsOverlayScrollbarEnabled()) {
-    settings.scrollbar_animator = cc::LayerTreeSettings::Thinning;
+    settings.scrollbar_animator = cc::LayerTreeSettings::LinearFade;
   }
   if (cmd->HasSwitch(cc::switches::kEnablePinchVirtualViewport) ||
       IsOverlayScrollbarEnabled()) {
     settings.solid_color_scrollbar_color = SkColorSetARGB(128, 128, 128, 128);
   }
 #endif
 
   compositor->Initialize(settings);
